apps/dynatrace/perftest/base/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../../base
+  - ../../dynatrace-operator2.yaml
+  - ../../dynatrace-operator-helmrepo2.yaml
+namespace: dynatrace
```

Comment:
Similar to a comment below - new one is being added to both clusters from here, but then you're only adding a patch for 01. Can you review please

Reply:
Added patch for 00, I was orig only going to test against 01.